### DISCUSSION TITLE
Unity personal

### DISCUSSION THREAD

**Discussion by FrixQn** [codemagic-ci-cd/codemagic-docs#2480] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Unity

### Steps to reproduce

How can I activate a personal license to test the Unity build using code magic? 
I tried using the -force-free command, but this does not solve my problem and a message is displayed stating that there is no valid license. 
What if I don't have it in principle?

### Expected results

If you try to activate the force-free license by logging into your account, the license service considers that the license is invalid. But I need the opportunity to test the build from the beginning on a personal version

### Actual results

If you try to activate the force-free license by logging into your account, the license service considers that the license is invalid

**Comment by dtrdic** [comment]
Hi @FrixQn please note that building Unity apps in a cloud CI/CD environment requires a Unity Plus or a Pro license

**Reply by wilg** [reply]
This does not appear to be a Unity licensing requirement as far as I can tell.

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
